<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>100</x>
    <y>100</y>
    <width>360</width>
    <height>320</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
   <widget class="QWidget" name="centralwidget">
    <property name="styleSheet">
     <string notr="true">
QLineEdit {
    min-height: 32px;
    border-radius: 2px;
    border-style: solid;
    border-width: 1px;
    border-color: rgba(0, 0, 0, 64);
    margin-top: 4px;
    margin-bottom: 4px;
}

QLineEdit:focus {
    border-color: rgba(0, 147, 196, 217);
}

QLineEdit:hover {
    border-color: rgba(0, 0, 0, 102);
}

QLineEdit:disabled {
    border-color: rgba(0, 0, 0, 64);
    background-color: rgba(255, 255, 255, 128);
    color: rgba(0, 0, 0, 128);
}
     </string>
    </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_title">
      <property name="text">
       <string>Login User</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel { font-size:20px; font-weight:bold; }</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_username">
        <property name="text">
         <string>Username</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel { font-size:18px; }</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_username"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_password">
        <property name="text">
         <string>Password</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel { font-size:18px; }</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_password">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_role">
        <property name="text">
         <string>Role</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel { font-size:18px; }</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBox_role">
        <item><property name="text"><string>Senior Tester</string></property></item>
        <item><property name="text"><string>Developer</string></property></item>
        <item><property name= "text"><string>Senior Developer</string></property></item>
        <item><property name= "text"><string>Tester</string></property></item>
       </widget>
      </item>
     </layout>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="pushButton">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>28</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>35</height>
       </size>
      </property>
      <property name="text"><string>Login</string></property>
      <property name="styleSheet"><string notr="true">QPushButton { font-size:15px; }</string></property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label_loginlink">
      <property name="text"><string>Don't have an account? &lt;a href="register"&gt;Click here&lt;/a&gt;</string></property>
      <property name="textFormat"><enum>Qt::RichText</enum></property>
      <property name="textInteractionFlags"><set>Qt::TextBrowserInteraction</set></property>
      <property name="openExternalLinks"><bool>false</bool></property>
     </widget>
    </item>
    
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>